Gems/Compression/Code/Platform/Linux/PAL_linux.cmake

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+set(PAL_TRAIT_COMPRESSION_SUPPORTED TRUE)
+set(PAL_TRAIT_COMPRESSION_TEST_SUPPORTED TRUE)
+set(PAL_TRAIT_COMPRESSION_EDITOR_TEST_SUPPORTED TRUE)
```

Comment:
nit: Add newline
(Repeated in similar files)

Reply:
The Gem template itself is missing the new newlines.